config/manager/manager.yaml

```diff
@@ -73,4 +73,11 @@ spec:
         args:
         - "controller"
         - --v={{.Values.controller.verbosity}}
-        - --agent-v={{.Values.agent.verbosity}}
+        - "{{if .Values.agent.hub_burst}}--agent-hub-burst={{.Values.agent.hub_burst}}{{end}}"
+        - "{{if .Values.agent.hub_qps}}--agent-hub-qps={{.Values.agent.hub_qps}}{{end}}"
+        - "{{if .Values.agent.local_burst}}--agent-local-burst={{.Values.agent.local_burst}}{{end}}"
+        - "{{if .Values.agent.local_qps}}--agent-local-qps={{.Values.agent.local_qps}}{{end}}"
+        - "{{if .Values.agent.log_flush_frequency}}--agent-log-flush-frequency={{.Values.agent.log_flush_frequency}}{{end}}"
+        - "{{if .Values.agent.logging_format}}--agent-logging-format={{.Values.agent.logging_format}}{{end}}"
+        - "{{if .Values.agent.v}}--agent-v={{.Values.agent.v}}{{end}}"
+        - "{{if .Values.agent.vmodule}}--agent-vmodule={{.Values.agent.vmodule}}{{end}}"
```

Comment:
I am not entirely happy to duplicate this much specific knowledge here and in values.yaml. The good news is that I expect that changes in the logging flags will be rare (based on historical experience). There is another approach possible: https://github.com/kubestellar/ocm-status-addon/issues/57#issuecomment-2128612880